kubernetes/hmda-platform/README.md

```diff
@@ -0,0 +1,30 @@
+Pre-requisites
+- [k8ssandra](https://k8ssandra.io/)
+- [Strimzi](https://strimzi.io/)
+
+Install
+- Secrets
+```
+kubectl create secret generic cassandra-credentials --from-literal=cassandra.username=  --from-literal=cassandra.password=
+kubectl create secret generic inst-postgres-credentials --from-literal=username= --from-literal=password= --from-literal=host= --from-literal=url="jdbc:postgresql://postgresql:5432/hmda?user= &password= &sslmode=false"
+```
+- Configmap
+```
+kubectl apply -f https://github.com/cfpb/hmda-platform/tree/master/kubernetes/config-maps
+```
+
+Update
+```
+helm upgrade --install --force --namespace=default --values=kubernetes/hmda-platform/values-dev.yaml \
```

Comment:
I'd probably remove `--force` and see the issues before deciding to "force" it; if there are issues.